apps/lockfile-explorer/src/start.ts

```diff
@@ -19,6 +19,8 @@ app.use(cors());
 
 app.use('/app', express.static(path.resolve(__dirname, '../dist')));
 
+app.use('/favicon.ico', express.static(path.resolve(__dirname, '../dist'), { index: 'favicon.ico' }));
```

Comment:
(The browser looks for `//localhost/favicon.ico` not `//localhost/app/favicon.ico`.)